playwright/_impl/_driver.py

```diff
@@ -16,17 +16,18 @@
 import os
 import sys
 from pathlib import Path
+from typing import Tuple
 
 import playwright
 from playwright._repo_version import version
 
 
-def compute_driver_executable() -> Path:
-    package_path = Path(inspect.getfile(playwright)).parent
-    platform = sys.platform
-    if platform == "win32":
-        return package_path / "driver" / "playwright.cmd"
-    return package_path / "driver" / "playwright.sh"
+def compute_driver_executable() -> Tuple[str, str]:
+    driver_path = Path(inspect.getfile(playwright)).parent / "driver"
+    cli_path = str(driver_path / "package" / "cli.js")
+    if sys.platform == "win32":
+        return (str(driver_path / "node.exe"), cli_path)
+    return (os.getenv("PLAYWRIGHT_NODEJS_PATH", str(driver_path / "node")), cli_path)
```

Comment:
`PLAYWRIGHT_NODEJS_PATH` is something new. Stray change?

Reply:
This is something we handled before in the wrapper script: https://github.com/microsoft/playwright/blob/c712b365ad7a180a2e7106c69fce8117f5b85c75/utils/build/run-driver-posix.sh#L3